<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ajouter_equipe</class>
 <widget class="QDialog" name="Ajouter_equipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Ajouter_equipe_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Appliquer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Categorie:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_image">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>image</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_nom">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>37</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_categorie">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>77</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_manager">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ajout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>391</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>41</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom:</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>123</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Manager:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>185</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total_joueur:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="in_number">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>265</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="in_score">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <zorder>label_ajout</zorder>
  <zorder>Ajouter_equipe_2</zorder>
  <zorder>label_2</zorder>
  <zorder>b_image</zorder>
  <zorder>in_nom</zorder>
  <zorder>in_categorie</zorder>
  <zorder>in_manager</zorder>
  <zorder>label</zorder>
  <zorder>image_2</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_7</zorder>
  <zorder>in_number</zorder>
  <zorder>label_8</zorder>
  <zorder>in_score</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
